corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserMain.java

```diff
@@ -98,28 +84,25 @@ public static void main(String[] args) {
             log.info("Successfully connected to {}", singleNodeEndpoint);
 
             CorfuStoreBrowser browser = new CorfuStoreBrowser(runtime);
-            CorfuTable<CorfuDynamicKey, CorfuDynamicRecord> table;
-            String namespace = opts.containsKey("--namespace") ?
-                    opts.get("--namespace").toString() : null;
-            String tableName = opts.containsKey("--tablename") ?
-                    opts.get("--tablename").toString() : null;
+            String namespace = Optional.ofNullable(opts.get("--namespace"))
+                    .map(n -> n.toString())
```

Comment:
will this not throw an NPR if namespace is null?

Reply:
No, that's why we use the ofNullable call right?